uid-pages/form-case-overview/src/caseoverview.json

```diff
@@ -1072,19 +1073,14 @@
       "value" : "../{{caseOverview.selectedBusinessDataRef.link}}",
       "exposed" : false
     },
-    "businessDataHeaders" : {
-      "type" : "expression",
-      "value" : "return $data.businessDataKeys.map(function(key){\n    return key.replace(\"_string\",\"\");\n});",
-      "exposed" : false
-    },
     "case" : {
       "type" : "expression",
       "value" : "return  (!$data.openCase && $data.archivedCase.length > 0) ? $data.archivedCase[0] : $data.openCase;",
       "exposed" : false
     },
     "businessDataKeys" : {
       "type" : "expression",
-      "value" : "var businessData = $data.selectedBusinessData.length ? $data.selectedBusinessData[0] :  $data.selectedBusinessData;\n\nreturn Object.keys(businessData).filter(function(header){\n    return  header  !== \"persistenceId\" \n            && header.indexOf(\"persistenceVersion\") === -1 \n            && header !== \"links\";\n});",
+      "value" : "function objectIdAsString(header, headers) {\n    return header.endsWith(\"_string\") && headers.includes(header.replace(\"_string\", \"\"));\n}\n\nvar businessData = $data.selectedBusinessData.length ? $data.selectedBusinessData[0] :  $data.selectedBusinessData;\n\nreturn Object.keys(businessData).filter(function(header, index, headers){\n    return  !objectIdAsString(header, headers);\n}).filter(function(header){\n    return  header !== \"persistenceId\"\n            && header.indexOf(\"persistenceVersion\") === -1 \n            && header !== \"links\" \n});",
```

Comment:
Adding a test to validate:
- all the required header are displayed 
- we do not filter header like myheader_string

Reply:
done